packages/smart-wallet/test/swingsetTests/upgradeWalletFactory/walletFactory-V2.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @file fixture for an upgrade of the primary walletFactory contract,
+ * packages/smart-wallet/src/walletFactory.js
+ *
+ * This variant has minor changes to the returned strings that make it
+ * identifiable, to demonstrate that upgrade has occurred.
+ */
+
+import { M, makeExo, mustMatch } from '@agoric/store';
+import { makeAtomicProvider } from '@agoric/store/src/stores/store-utils.js';
+import { makeScalarBigMapStore } from '@agoric/vat-data';
+import { E } from '@endo/far';
+import { prepareSmartWallet } from '../../../src/smartWallet.js';
+import { shape } from '../../../src/typeGuards.js';
+import {
+  makeAssetRegistry,
+  publishDepositFacet,
+} from '../../../src/walletFactory.js';
+
+/**
+ *
+ * @type {typeof import('../../../src/walletFactory.js').start}
+ */
+export const start = async (zcf, privateArgs, baggage) => {
```

Comment:
upgradeable contracts export `prepare` rather than `start`. Did the test pass?
```suggestion
export const prepare = async (zcf, privateArgs, baggage) => {
```

Reply:
no, early draft here